plugins/inputs/logparser/logparser.go

```diff
@@ -159,6 +160,7 @@ func (l *LogParserPlugin) tailNewfiles(fromBeginning bool) error {
 					Follow:    true,
 					Location:  &seek,
 					MustExist: true,
+					Poll:      true, //inotify(default) not work in macos(darwin). poll test ok.
```

Comment:
What we could do here is allow this to be configurable.  We can add an option to the plugin called `watch_method` with available options `"poll"` and `"inotify"`.  If unset, we could automatically pick an implementation.